pubspec.yaml

```diff
@@ -1,15 +1,13 @@
 name: xrange
-version: 1.0.0
+version: 1.1.0-nullsafety.0
```

Comment:
@hovadur could you please increase the major version instead of the minor one:

```
version: 2.0.0-nullsafety.0
```

Reply:
done